db.js

```diff
@@ -435,7 +444,25 @@ export class ActivityPubDB extends EventTarget {
     console.log(note.replies)
     if (note.replies && typeof note.replies === 'string') {
       console.log('Attempting to load replies for:', note.id)
-      await this.ingestActivityCollection(note.replies, note.attributedTo, true)
+      try {
+        await this.ingestReplies(note.replies)
+      } catch (error) {
+        console.error(`Failed to ingest replies for ${note.id}:`, error)
+      }
+    }
+  }
+
+  async ingestReplies (url) {
+    console.log('Ingesting replies for URL:', url)
+    try {
+      const repliesCollection = await this.#get(url)
```

Comment:
No need for this or the if statement, `iterateCollection` does the get and checking items for us already.

just keep `this.iterateCollection(url, {limit: Infinity})`